.eslintrc

```diff
@@ -1,8 +1,12 @@
 {
-    "ecmaFeatures": {
-        "globalReturn": true,
-        "jsx": true,
-        "modules": true
```

Comment:
ESLint throws an error during webpack build if "sourceType" is not set. (v 2.0.8)